web/wp-content/themes/dctx-v2/template-parts/content-search.php

```diff
@@ -0,0 +1,30 @@
+<?php
```

Comment:
:no_entry_sign: **Error**: Filenames should be all lowercase with hyphens as word separators. Expected phpcs-scan-onqtz6.php, but found phpcs-scan-onqTZ6.php (*WordPress.Files.FileName.NotHyphenatedLowercase*).